jasmine/spec/inverted-index-test.js

```diff
@@ -1,46 +1,78 @@
 var index = new Index();
 describe('Read book data', function() {
-    it('Asserts that json file is not empty', function(done) {
-        // Resolving the promise from the createIndex method 
-        // Data is the a Javascript object with the Json data
-        index.fetchJSON('../jasmine/books.json').then(function(data) {
-            expect(data).toBeDefined();
-            expect(typeof data).toEqual('object');
-            expect(data.length).toBeGreaterThan(0);
-            done();
-        });
+
+  it('Asserts that JSON file is not empty', function(done) {
+    // Resolve the promise from fetchJSON method
+    // Data is the Javascript object with JSON data
+    // 
+    index.fetchJSON('../jasmine/books.json').then(function (data) {
+      expect(data).toBeDefined();
+      expect(typeof data).toEqual('object');
+      expect(data.length).toBeGreaterThan(0);
+      done();
     });
+
+  });
+  
 });
```

Comment:
@andela-mkwamboka This was not amongst the comments. I  think you misread my intention.